packages/CollabCodeEdit-Client.package/CCEModel.class/instance/initialize.st

```diff
@@ -1,13 +1,16 @@
 initialize-release
 initialize
 	super initialize.
-	SystemBrowser default: CCEBrowser.
 	self createSessionModel: CCECreateSessionModel new;
 		account: CCEAccount default;
 		joinSessionModel: CCEJoinSessionModel new;
 		leaveSessionModel: CCELeaveSessionModel new;
-		client: (CCEClient default owner: self);
-		session: (CCESession default client: self client).
+		client: (CCEClient default 
+				activeUI: self;
```

Comment:
These lines should only be indented by 1 additional level

Reply:
They are.